### PR TITLE
Add new players for venom tests

### PR DESCRIPTION
**Pull request by JeGoi**
# Description
Add new players

# Impacts
New players for venom tests

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Need to add psonos stuff according to documentation

### CODE REVIEW

File: addons/vagrant/inventory/hosts [hunk 1]
```diff
@@ -1671,3 +1671,329 @@ all:
             netmask: '255.255.255.0'
             ip_reg: '172.18.156.34'
             ip_iso: '172.18.157.34'
+      _abprasa:
+        vars:
+          rhel_sub_secret_id: ''
```
Comment (abprasa) [inline addons/vagrant/inventory/hosts]:
```suggestion
          rhel_sub_secret_id: '05e7cadd-ea00-4d98-8097-60c14c674afc'
```

File: addons/vagrant/inventory/hosts [hunk 2]
```diff
@@ -1671,3 +1671,329 @@ all:
...
+          fingerbank_api_key:
+            secret_id: ''
```
Comment (abprasa) [inline addons/vagrant/inventory/hosts]:
```suggestion
            secret_id: 'caf645c1-4134-46f2-9e6e-5469e0274b7e'
```

File: addons/vagrant/inventory/hosts [hunk 3]
```diff
@@ -1671,3 +1671,329 @@ all:
...
+            email: ''
```
Comment (abprasa) [inline addons/vagrant/inventory/hosts]:
```suggestion
            email: 'abprasa@akamai.com'
```

File: addons/vagrant/inventory/hosts [hunk 4]
```diff
@@ -1671,3 +1671,329 @@ all:
...
+            ip_reg: '172.18.161.33'
+            ip_iso: '172.18.162.33'
+          pf3deb11localdev:
+            ip: '172.18.160.34'
+            netmask: '255.255.255.0'
+            ip_reg: '172.18.161.34'
+            ip_iso: '172.18.162.34'
+      _zma:
+        vars:
+          rhel_sub_secret_id: ''
```
Comment (stgmsa) [inline addons/vagrant/inventory/hosts]:
```suggestion
          rhel_sub_secret_id: 'bb5ea639-b4aa-4c93-b86f-25d14c80bb53'
```

File: addons/vagrant/inventory/hosts [hunk 5]
```diff
@@ -1671,3 +1671,329 @@ all:
...
+          fingerbank_api_key:
+            secret_id: ''
```
Comment (stgmsa) [inline addons/vagrant/inventory/hosts]:
```suggestion
            secret_id: 'eb7a9c9b-545e-41b0-90d3-80b6a96b742c'
```

File: addons/vagrant/inventory/hosts [hunk 6]
```diff
@@ -1671,3 +1671,329 @@ all:
...
+            email: ''
```
Comment (stgmsa) [inline addons/vagrant/inventory/hosts]:
```suggestion
            email: 'zma@akamai.com'
```

**Review by stgmsa — COMMENTED** [review]
mine configurations looks good

File: addons/vagrant/inventory/hosts [hunk 7]
```diff
@@ -1671,3 +1671,329 @@ all:
             netmask: '255.255.255.0'
             ip_reg: '172.18.156.34'
             ip_iso: '172.18.157.34'
+      _abprasa:
```
Comment (nqb) [inline addons/vagrant/inventory/hosts:1674]: Must be to follow naming convention:
```suggestion
      _aprasad:
```

Comment (nqb) [reply addons/vagrant/inventory/hosts:1674]: and also in next lines.

Comment (JeGoi) [reply addons/vagrant/inventory/hosts:1674]: Like discussed it is more the akamai convention way.
Lets keep it

**Pull request merged**